packages/validator/docs/dynamic_params.md

```diff
@@ -0,0 +1,183 @@
+# Dynamic values and keys
+
```

Comment:
Add something like:
> The validator template accepts both dynamic keys and dynamic values. Dynamic parameters contain placeholders that are filled with strings contained in other parts of the template.

Reply:
That seems a bit confusing, I added this:

> The validator can access both, keys from the parameter path and values from the specification. These dynamic parameters consist of placeholders that are on evaluation replaced with strings contained in other parts of the specification.

Reply:
sure! just one thing: i think the first comma is unnecessary.